skimage/restoration/deconvolution.py

```diff
@@ -153,20 +153,40 @@ def unsupervised_wiener(
     Parameters
     ----------
     image : (M, N) ndarray
-       The input degraded image.
+        The input degraded image.
     psf : ndarray
-       The impulse response (input image's space) or the transfer
-       function (Fourier space). Both are accepted. The transfer
-       function is automatically recognized as being complex
-       (``np.iscomplexobj(psf)``).
+        The impulse response (input image's space) or the transfer
+        function (Fourier space). Both are accepted. The transfer
+        function is automatically recognized as being complex
+        (``np.iscomplexobj(psf)``).
     reg : ndarray, optional
-       The regularisation operator. The Laplacian by default. It can
-       be an impulse response or a transfer function, as for the psf.
+        The regularisation operator. The Laplacian by default. It can
+        be an impulse response or a transfer function, as for the psf.
     user_params : dict, optional
-       Dictionary of parameters for the Gibbs sampler. See below.
-    clip : boolean, optional
-       True by default. If true, pixel values of the result above 1 or
-       under -1 are thresholded for skimage pipeline compatibility.
+        Dictionary of parameters for the Gibbs sampler. Accepted keys are:
+
+        threshold : float
+           The stopping criterion: the norm of the difference between to
+           successive approximated solution (empirical mean of object
+           samples, see Notes section). 1e-4 by default.
+        burnin : int
+           The number of sample to ignore to start computation of the
+           mean. 15 by default.
+        min_num_iter : int
+           The minimum number of iterations. 30 by default.
+        max_num_iter : int
+           The maximum number of iterations if ``threshold`` is not
+           satisfied. 200 by default.
+        callback : callable (None by default)
```

Comment:
Here is a parenthetical comment; is that OK in this context? I saw you replaced all above.

Reply:
Yes, kind of. This is an indented list (can't remember what sphinx calls these) inside the description section of the `user_params` parameter. It's not picked up by NumPyDoc and therefore also isn't parsed by docstub.

But doesn't hurt to make this consistent.

I restructured the docstring here, because it was using a custom "Other parameters" section which was flagged by NumPyDoc as a warning. The proposed formatting is something I've seen in SciPy and [renders nicely](https://output.circle-artifacts.com/output/job/1162ef14-e6c4-4ed2-b2df-a485947db8ce/artifacts/0/doc/build/html/api/skimage.restoration.html#skimage.restoration.unsupervised_wiener:~:text=user_paramsdict%2C%20optional,Accepted%20keys%20are%3A) I think.